float32 altitude
float32 altitude_filt
float32 altitude_der